solr/core/ivy.xml

```diff
@@ -138,6 +138,9 @@
     <dependency org="com.google.protobuf" name="protobuf-java" rev="${/com.google.protobuf/protobuf-java}" conf="compile"/>
     <dependency org="com.jayway.jsonpath" name="json-path" rev="${/com.jayway.jsonpath/json-path}" conf="compile"/>
 
+    <!-- Package manager -->
```

Comment:
what is this for?

Reply:
This is for semantic versions (https://semver.org/). This library gives us comparison and range validations of versions.